### PR TITLE
docs: add blog mysqlToSlack

### PR DESCRIPTION
**Pull request by Michaelg22**
Signed-off-by: Michaelg22 <mike.giroux22@outlook.com>

### Related Issue

Solving Issue: #xxx

### PR Type

- [ ] Fix typos or format(punctuation, space, indentation, code block, etc.)
- [ ] Update inappropriate or misunderstanding content
- [ ] Add missing content or new documents
- [ ] Delete useless content or outdated documents
- [ ] Localize or translate documents
- [ ] Deal with GitHub action files and scripts
- [ ] Deal with chores

### CODE REVIEW

File: blog/2023-02-04-mysql-slack-blog/index.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+To achieve that, we need one program to receive database events and transform them into custom messages and another to receive transformed messages and send them to our Slack channel.
+
+Luckily, Vanus has already provided many ***out-of-the-box Connectors*** that enable you to integrate with popular services or applications without writing code.
+
+***Prerequisites***
+- Have MySQL Server 8.0, 5.7, or 5.6.
+- Have a Slack account.
+
+Here is how to do this in only 10 mins.
+1. Go to Vanus Playground (an online K8s playground)
```
Comment (ehisakhile) [inline blog/2023-02-04-mysql-slack-blog/index.md:24]: Add Playground URL 
https://play.linkall.com

File: blog/2023-02-04-mysql-slack-blog/index.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+          configMap:
+            name: sink-slack
+EOF
+```
+
+21. Update the settings: `Channel` and `token`
+
+```shell
+vi slack-sink.yml
+```
```
Comment (ehisakhile) [inline blog/2023-02-04-mysql-slack-blog/index.md:204]: Need to add the command to run slack-sink.yml
kubectl apply -f slack-sink.yml

File: blog/2023-02-04-mysql-slack-blog/index.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+  `password` varchar(100) NOT NULL,
+`subscription_plan` varchar(100) NOT NULL,
+`phone_number` varchar(100) NOT NULL,
+
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+```
+
+```shell
+Create a subscription ->
```
Comment (ehisakhile) [inline blog/2023-02-04-mysql-slack-blog/index.md:229]: Create a subscription should not be part of the code

File: blog/2023-02-04-mysql-slack-blog/index.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+vsctl subscription create --name mysql-sub \
```
Comment (ehisakhile) [inline blog/2023-02-04-mysql-slack-blog/index.md:230]: vsctl subscription create \

vsctl subscription create --name mysql-sub \

File: blog/2023-02-04-mysql-slack-blog/index.md [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+  --eventbus mysql-to-slack \
+  --sink 'http://sink-slack:8080' \
+  --transformer '{
+      "define": { # define your own veriable
```
Comment (ehisakhile) [inline blog/2023-02-04-mysql-slack-blog/index.md:234]: Remove comment tag, (# define your own variable)
Makes transformer invalid. Instead, add as comment under code

File: blog/2023-02-04-mysql-slack-blog/index.md (hunk 2 above)
Comment (Michaelg22) [reply blog/2023-02-04-mysql-slack-blog/index.md:204]: thanks

File: blog/2023-02-04-mysql-slack-blog/index.md (hunk 3 above)
Comment (Michaelg22) [reply blog/2023-02-04-mysql-slack-blog/index.md:229]: I have a reason for doing this

File: blog/2023-02-04-mysql-slack-blog/index.md (hunk 4 above)
Comment (Michaelg22) [reply blog/2023-02-04-mysql-slack-blog/index.md:230]: yup version 0.6

File: blog/2023-02-04-mysql-slack-blog/index.md (hunk 5 above)
Comment (Michaelg22) [reply blog/2023-02-04-mysql-slack-blog/index.md:234]: ok

File: blog/2023-02-04-mysql-slack-blog/index.md [hunk 6]
```diff
@@ -0,0 +1,246 @@
...
+        "subscription_plan": "$.data.subscription_plan",
+        "phone_number": "$.data.phone_number"
+      },
+      "template": "{\"subject\":\"sales team\",\"message\":\"Hi Sales team, We have a new user <${username}>, Email: <${email}>, Subscription plan: <${subscription_plan}>, Phone Number: ${phone_number}.\"}"
+    }'
+```
+23. Let's test everything by making an entry in MySQL.
+
+```mysql
+INSERT into DATABASE.TABLE values(1,"jess","mypassword","Email@jesica.com","sub 1 month","434 555 1232");
```
Comment (ehisakhile) [inline blog/2023-02-04-mysql-slack-blog/index.md:246]: INSERT into mybmax2.customer values(1,"jess","mypassword","Email@jesica.com","sub 1 month","434 555 1232");

Comment (Michaelg22) [reply blog/2023-02-04-mysql-slack-blog/index.md:252]: ?

File: blog/2023-02-04-mysql-slack-blog/index.md (hunk 3 above)
Comment (ehisakhile) [reply blog/2023-02-04-mysql-slack-blog/index.md:235]: It throws errors on Playground

File: blog/2023-02-04-mysql-slack-blog/index.md (hunk 6 above)
Comment (ehisakhile) [reply blog/2023-02-04-mysql-slack-blog/index.md:252]: Nevermind

File: blog/2023-02-04-mysql-slack-blog/index.md (hunk 3 above)
Comment (ehisakhile) [reply blog/2023-02-04-mysql-slack-blog/index.md:235]: Add # to make it a comment

File: blog/2023-02-04-mysql-slack-blog/index.md [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+  `password` varchar(100) NOT NULL,
+`subscription_plan` varchar(100) NOT NULL,
+`phone_number` varchar(100) NOT NULL,
+
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB;
+```
+
+```shell
+Create a subscription ->
```
Comment (Michaelg22) [inline blog/2023-02-04-mysql-slack-blog/index.md:235]:
```suggestion
# Create a subscription ->
```

**Review by ehisakhile — APPROVED** [review]

**Review by VedaGao — APPROVED** [review]

**Pull request merged**